djstripe/models.py

```diff
@@ -593,6 +563,39 @@ def record_charge(self, charge_id):
         data = stripe.Charge.retrieve(charge_id)
         return Charge.sync_from_stripe_data(data)
 
+    @staticmethod
+    @webhook.handler_all
+    def event_attach_customer(event, data, cat, sub_cat):
+        stripe_customer_crud_events = ["created", "updated", "deleted", ]
+        skip_events = ["plan", "transfer", ]
```

Comment:
You're correct about multi-line lists, but the convention for single line lists in dj-stripe is to drop the trailing comma because it looks cleaner. It's not a strong preference per se, but I'd prefer to keep consistency.